### PR TITLE
feat(skills): support --force on skill ZIP upload from console

### PR DESCRIPTION
**Pull request by snoller**
Adds a `force` query param to `POST /api/admin/skills/upload` that lets the gateway overwrite an existing skill instead of returning 409. The console exposes the option as an "Overwrite existing skill (--force)" checkbox under the ZIP file picker, mirroring the existing `hybridclaw skill import --force` CLI behavior.

Also nudges the spacing above the create-skill form so the toggle no longer sits flush against "Name"/"Skill archive (.zip)".

## Summary

Describe the change in 2-5 bullets:

- Problem:
- Why it matters:
- What changed:
- What did not change:

## Change Type

- [ ] Bug fix
- [ ] Feature
- [ ] Docs
- [ ] Tests
- [ ] Refactor required for the fix
- [ ] Tooling or workflow
- [ ] Security hardening

## Linked Context

- Closes #
- Related #

## Manifesto Principle

Which manifesto principle does this serve?

- Principle:
- Changelog tag added or updated: `Manifesto: Principle <Roman numeral> - <principle title>.`

## Validation

List the checks you ran and the concrete scenarios you verified.

```bash
# Example
npm run typecheck
npm run lint
npm run test:unit
```

- Verified manually:
- Edge cases checked:
- Skipped checks and why:

## Docs And Config Impact

- [ ] README, docs, or examples updated
- [ ] Config or environment behavior changed
- [ ] Templates or workspace bootstrap files changed
- [ ] No docs or config impact

If `templates/` changed, confirm whether `src/workspace.ts` and the related
tests were updated.

## Risk Notes

- Security-sensitive paths touched? (`Yes/No`)
- Gateway, audit, approval, or container boundaries touched? (`Yes/No`)
- If yes, what is the failure mode and how did you test it?

## Evidence

Attach at least one when relevant:

- [ ] Failing output before and passing output after
- [ ] Screenshot or recording
- [ ] Log snippet
- [ ] New or updated test coverage

### CODE REVIEW

File: src/gateway/gateway-service.ts [hunk 1]
```diff
@@ -5929,12 +5930,16 @@ export async function uploadGatewayAdminSkillZip(
...
+    const targetExists = fs.existsSync(targetDir);
+    if (targetExists && !options.force) {
       throw new GatewayRequestError(
         409,
         `Skill \`${skillName}\` already exists at ${targetDir}.`,
       );
     }
+    if (targetExists) {
+      fs.rmSync(targetDir, { recursive: true, force: true });
+    }
```
[the remark this answers is not in the document]
Comment (furukama) [reply src/gateway/gateway-service.ts]: @snoller Check this comment please
